frontend/src/lib/i18n/translations/en/candidateApp.common.json

```diff
@@ -2,10 +2,11 @@
   "contactSupport": "Contact Support",
   "continueFilling": "Continue Filling",
   "editingNotAllowed": "Editing of your answers is no longer possible, but you can still view them. Please contact the administrator if there are some errors in the data.",
-  "editingNotAllowedPartiallyFilled": "Because you didn’t answer all of the questions, your profile won’t be shown.",
```

Comment:
This now becomes conditional on the relevant setting. See the next translation file for the logic.